LocalSettings.php

```diff
@@ -5708,6 +5709,19 @@
 		'fbwikiwiki' => array('visualeditor'),
 		'spiralwiki' => array('visualeditor'),
 	),
+	
+	// Protect site config
+	'wgProtectSiteLimit' => array(
+		'default' => '1 week',
+		'tnoteswiki' => 'indefinite',
+	),
+	'wgProtectSiteDefaultTimeout' => array(
+		'default' => '1 hour',
+		'tnoteswiki' => '2 hours',
+	),
+	'wgProtectSiteExempt' => array(
```

Comment:
I'm not sure if we must have these values, but if we have custom values for extension variables, I'd prefer that we have the default setting listed too.